src/Utility/External/SortedList.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * SortedList Object. Sorted Array of element (ascending or descending order)
+ *
+ * @author KhaaZ
+ *
+ * @template T
+ *
+ * @class SortedList
+ * @prop {Boolean} acending
+ * @prop {(toInsert, baseElement) => Boolean)} comparator - Wether toInsert is bigger than baseElement
+ * @prop {Array<T>} list
+ */
+class SortedList {
+    constructor(comparator = null, ascending = true) {
+        console.log(comparator);
+        this.comporator = comparator || ( (toInsert, baseElement) => toInsert >= baseElement);
```

Comment:
```suggestion
        this.comparator = comparator || ( (toInsert, baseElement) => toInsert >= baseElement);
```
khaaz is having a moment lol

Reply:
Apparently I also used `comporotor` in  add lol 💯